### PR TITLE
📖 Update packaging.md

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
## Summary
This PR does various corrections, extensions, and updates to packaging.md.

## Related issue(s)

Fixes #

### CODE REVIEW

File: docs/content/v0.20/packaging.md [hunk 1]
```diff
@@ -12,11 +12,14 @@ There is also a container image involved.
...
 ### Local copy of KubeStellar git repo
 
-Because of [a restriction in one of the code generators that we
+**NOTE**: Because of [a restriction in one of the code generators that
+we
 use](https://github.com/kubernetes/code-generator/blob/v0.28.2/kube_codegen.sh#L394-L395),
-a contributor who works on API changes needs to have their local copy
-of the git repo in a directory whose name ends with the Go package
-name --- that is, ends with `github.com/kubestellar/kubestellar/`.
+a contributor who works on API changes, or even just invokes
```
Comment (ezrasilvera) [inline docs/content/v0.20/packaging.md:18]: Should we explain all of this or just recommend this path for all developers?  Especially we add "or even just invokes.."   without explaining why and when a developer may need to run this.

File: docs/content/v0.20/packaging.md [hunk 2]
```diff
@@ -47,19 +51,40 @@ The files in [pkg/generated](../../../pkg/generated) are generated by [k/code-ge
 
 ### Central container image
 
-Currently at `ghcr.io/kubestellar/kubestellar/kubestellar-operator:0.20.0-alpha.1`.
+KubeStellar has one container image, for what is called the
+KubeStellar operator. This operator installs the needed custom
+resource _definition_ objects if they are not already present, and is
+a controller-manager hosting the relevant central controllers.
```
Comment (ezrasilvera) [inline docs/content/v0.20/packaging.md:57]: It's not clear what are those "relevant central controllers" . So I think we should either explain + examples , or remove it.

File: docs/content/v0.20/packaging.md [hunk 3]
```diff
@@ -47,19 +51,40 @@ The files in [pkg/generated](../../../pkg/generated) are generated by [k/code-ge
...
+`ghcr.io/kubestellar/kubestellar/kubestellar-operator`. There is
+currently just one supported tag there, `0.20.0-alpha.1`. That was
+built from the git repo contents with the tag `v0.20.0-alpha.1`.
+
+`make ko-build-push` will build and push that image, tagging it with
+the value of the `make` variable `IMAGE_TAG` (which defaults to
+`0.20.0-alpha.1`).
+
+`make ko-build-local` will make a local image for just the local
+platform.
```
Comment (ezrasilvera) [inline docs/content/v0.20/packaging.md:69]: Maybe add that this is used for local testing ?

File: docs/content/v0.20/packaging.md [hunk 4]
```diff
@@ -47,19 +51,40 @@ The files in [pkg/generated](../../../pkg/generated) are generated by [k/code-ge
...
 
 ### KubeStellar Operator Helm Chart
 
-The source for a Helm chart is in [core-helm-chart](../../../core-helm-chart). `make chart` (re)derives it from local sources.
+There is a Helm chart that is designed to be instantiated in a KubeFlex hosting cluster, once per WDS. The focus of the chart is getting the KubeStellar Operator installed.
```
Comment (ezrasilvera) [inline docs/content/v0.20/packaging.md:73]: Why "KubeFlex"  hosting ?  Maybe just hosting ?

File: docs/content/v0.20/packaging.md (hunk 2 above)
Comment (MikeSpreitzer) [reply docs/content/v0.20/packaging.md:57]: I plan to update this later with a reference into the architecture document, which I expect will be filled out with more content and gain a section heading that will be a good target for the reference.

File: docs/content/v0.20/packaging.md (hunk 4 above)
Comment (MikeSpreitzer) [reply docs/content/v0.20/packaging.md:73]: Let's fine-tune the wording here after the architecture document clearly defines the needed term.

File: docs/content/v0.20/packaging.md (hunk 1 above)
Comment (MikeSpreitzer) [reply docs/content/v0.20/packaging.md]: Done.

File: docs/content/v0.20/packaging.md (hunk 3 above)
Comment (MikeSpreitzer) [reply docs/content/v0.20/packaging.md]: Done.

**Pull request merged**